packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -175,4 +177,13 @@ export default abstract class EasCommand extends Command {
     await this.analytics.flushAsync();
     return super.finally(err);
   }
+
+  protected baseErrorMessage = 'Command failed';
+  protected override catch(err: CommandError): Promise<any> {
+    const cleanMessage = err.message.startsWith('[GraphQL] ')
```

Comment:
No strong opinion how to handle it, but maybe id you know that error is from grphql we should somehow mention that API request failed.
If you just want to strip leading graphql text, then this is fine, but if you decide to customize it more when handling graphql error, then I would suggest checking if `err.graphQLErrors` exists instead, instead of checking string prefix.

Reply:
@wkozyra95 Updated the check for graphQL error, and changed the base error message printed in that case